Content.Server/Chat/Commands/SuicideCommand.cs

```diff
@@ -86,8 +86,16 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
             //Checks to see if the CannotSuicide tag exits, ghosts instead.
             if(EntitySystem.Get<TagSystem>().HasTag(owner, "CannotSuicide"))
             {
-                EntitySystem.Get<GameTicker>().OnGhostAttempt(mind, false);
-                return;
+                if (!EntitySystem.Get<GameTicker>().OnGhostAttempt(mind, true))
```

Comment:
This function has side effects, it can only be run once, as it actually performs the ghosting.

Reply:
Should be fixed now